connect/go.sum

```diff
@@ -39,6 +39,8 @@ github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e/go.mod h1:nSuG5e5P
 github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1/go.mod h1:Q3SI9o4m/ZMnBNeIyt5eFwwo7qiLfzFZmjNmxjkiQlU=
 github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
 github.com/cncf/udpa/go v0.0.0-20191209042840-269d4d468f6f/go.mod h1:M8M6+tZqaGXZJjfX53e64911xZQV5JYwmTeXPW+k8Sc=
+github.com/confluentinc/ccloud-sdk-go-v2-internal/connect v0.0.6 h1:jgy7qVXB5HbY0tI5qRMQU1u4w0iWm/6LfSbel9YbO5w=
+github.com/confluentinc/ccloud-sdk-go-v2-internal/connect v0.0.6/go.mod h1:fyTZKsmAyKifi7HfAeM5goMpcjLIeASavL0r4bKt9+I=
```

Comment:
you might want to regenerate go.sum or remove these 2 lines

Reply:
Done